app/ecosystem/fixture.ts

```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
+    website: 'https://cobweblabs.xyz'
+  },
+  {
+    image: '/img/ecosystem/Tenderly.png',
+    title: 'Tenderly',
+    description:
+      'Full-stack Web3 infrastructure – node infra, exploration tools & dev environments.',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/TenderlyApp',
+    website: 'https://tenderly.co/'
+  },
+  {
+    image: '/img/ecosystem/Bithotel.jpg',
+    title: 'Bithotel',
+    description:
+      'Revolutionary social pixelart game (free-to-play!) on Mobile and PC.',
+    categories: ['gaming'],
+    twitter: 'https://x.com/playbithotel',
+    website: 'https://bithotel.io/#/'
+  },
+  {
+    image: '/img/ecosystem/TaskOn.svg',
+    title: 'TaskOn',
+    description:
+      'A Web3 task collaboration platform ideal for marketing, operations and community growth.',
+    categories: ['social'],
+    twitter: 'https://x.com/taskonxyz',
+    website: 'https://taskon.xyz/'
+  },
+  {
+    image: '/img/ecosystem/Questflow.jpg',
+    title: 'Questflow',
+    description:
+      'Decentralized Autonomous AI Agent Network - orchestrate multiple AI agents to take action on autopilot & dispatch real-world incentives to agents.',
+    categories: ['social'],
+    twitter: 'https://x.com/questflow',
+    website: 'https://www.questflow.ai/'
+  },
+  {
+    image: '/img/ecosystem/TheGraph.jpg',
+    title: 'The Graph',
+    description:
+      'A web3 protocol for organizing and accessing blockchain data.',
+    categories: ['infra'],
+    twitter: 'https://x.com/graphprotocol',
+    website: 'https://thegraph.com'
+  },
+  {
+    image: '/img/ecosystem/Omnisea.jpg',
+    title: 'Omnisea',
+    description: 'Omnichain NFT Launchpad powered by LayerZero.',
+    categories: ['nfts'],
+    twitter: 'https://x.com/omnisea',
+    website: 'https://omnisea.org/'
+  },
+  {
+    image: '/img/ecosystem/groupfi.jpg',
+    title: 'GroupFi',
+    description:
+      'A Web3 messaging protocol designed by the Tanglepay Team for seamless, feeless, on-chain and cross-application group communication. ',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/groupfi_ai',
+    website: 'https://groupfi.ai/'
+  },
+  {
+    image: '/img/ecosystem/ChessPlay.jpg',
+    title: 'Chess Play',
+    description:
+      'Chess meets blockchain. Your destination for innovative gameplay and secure transactions. Reshaping the future of on chain chess!',
+    categories: ['gaming'],
+    twitter: 'https://x.com/play_chess_/',
+    website: 'https://www.chessplay.live/'
+  },
+  {
+    image: '/img/ecosystem/Gamefi.jpg',
+    title: 'GameFi',
+    description: 'A One-stop Destination Launchpad For Web3 Enthusiasts.',
+    categories: ['launchpad'],
+    twitter: 'https://x.com/gamefi_official?lang=en',
+    website: 'https://gamefi.org/'
+  },
+  {
+    image: '/img/ecosystem/Openpad.jpg',
+    title: 'OpenPad',
+    description:
+      'Decentralized Crowdfunding Ecosystem + Onestop Incubation Hub for Web3 Projects.',
```

Comment:
```suggestion
      'Decentralized crowdfunding ecosystem + on-stop incubation hub for Web3 projects.',
```

"Web3" is capitalized here but not elsewhere.

Reply:
changed all to Web3